homeassistant/components/overkiz/coordinator.py

```diff
@@ -177,7 +177,7 @@ def _get_state(
         value = cast_to_python(state.value)
 
         if TYPE_CHECKING:
-            assert isinstance(value, (str, float, int, bool))
+            assert isinstance(value, (str, float, int, bool, dict, list))
```

Comment:
Is this TYPE_CHECKING still needed now that the underlying library is typed?

Reply:
Yes, I just double checked. 

```
homeassistant/components/overkiz/coordinator.py:182: error: Returning Any from function declared to return "Union[Dict[Any, Any], List[Any], float, int, bool, str, None]"  [no-any-return]
Found 1 error in 1 file (checked 13 source files)
```

This is related to this probably: https://github.com/home-assistant/core/blob/0c6c2910f88dd13a9994191e4ccc243ed4534e44/homeassistant/components/overkiz/coordinator.py#L27-L35

Reply:
Nevermind, found the culprit. Getting the hang of it. Good pointer!